content/docs/self-hosted-runners.md

```diff
@@ -452,17 +452,30 @@ The same credentials can also be used for
 
 The `cml runner` command can also be used to manually set up a local machine,
 on-premise GPU cluster, or any other cloud compute resource as a self-hosted
-runner. Simply [install CML](/doc/install) and then run:
+runner. To do this:
+
+- remove the `cml runner launch ... --cloud=aws --labels=cml-gpu` command from
```

Comment:
@iterative/websites any idea how to stop this from auto-hyperlinking to `/doc/ref/runner#--cloud`?

Reply:
moved to #424

Reply:
```suggestion
- remove the [`cml runner launch ... --cloud=aws --labels=cml-gpu`]() command from
```

Thanks to https://github.com/iterative/gatsby-theme-iterative/pull/153